mappings/net/minecraft/world/gen/WorldProvider.mapping

```diff
@@ -1,6 +1,8 @@
 CLASS none/auf net/minecraft/world/gen/WorldProvider
 	FIELD b world Lnone/aiw;
 	FIELD c biomeProvider Lnone/ajn;
+	FIELD d waterEvaporates Z
+	FIELD e noSky Z
```

Comment:
Hmm ok, it just feels weird to have a grammatically inverted boolean value I guess. Perhaps something more generic, like `underworld` might be better, especially since it's also used for world height.